app/konnect/org-management/saml-idp.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Set up SSO with SAML 2.0
```

Comment:
I'm not sure if we need this doc to be separate if we already cover this flow in the Okta doc.